lib/view.js

```diff
@@ -0,0 +1,73 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var URL = require('url');
+var assert = require('assert');
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = mixinView;
+
+function mixinView(Cloudant) {
+  var debug = require('debug')('loopback:connector:cloudant:view');
+
+  /**
+   * Gets data at `/{db}/_design/{ddocName}/views/{viewName}`
+   *
+   * Example:
+   * User has a view called `getModel` in design document /{db}/_design/model,
+   * to query the view, user can call function:
+   * ```
+   * ds.viewDocs(model, getModel, {key: 'purchase'}, cb);
+   * ```
+   *
+   * @param {String} ddocName The design doc name
```

Comment:
As a long time CouchDB user, _I am_ confused by whether I am expected to provide the whole name or not. Since the actual Couch/Cloudant document name is actually `_design/<name>` (with the `/` requiring URL encoding as `%2F`), it makes sense to _me_ to pass in the full name - unless I have written my own wrapper which adds it.

It is a case where knowing more means things are harder rather than easier.